src/pages/heyho/Login/Login.js

```diff
@@ -1,10 +1,55 @@
-import React from "react";
+import React from 'react';
 import { withRouter } from 'react-router-dom';
+import { Link } from 'react-router-dom';
+import './Login.scss';
+import InputId from './InputId';
```

Comment:
- 속성 순서 수정해주세요! 
`React → Library(Package) → Component → 변수 / 이미지 → css 파일(scss 파일)`

- react-router-dom 관련 import 는 다음과 같이 한 줄로 작성해 주실 수 있습니다.
```suggestion
import React from 'react';
import { Link, withRouter } from 'react-router-dom';
import InputId from './InputId';
import './Login.scss';
```